catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -110,7 +110,7 @@ public void initialize(
     resultConf.put("catalog_uuid", info.id().toString());
     IcebergConfig icebergConfig = new IcebergConfig(resultConf);
 
-    this.icebergCatalogWrapper = new IcebergCatalogWrapper(icebergConfig);
+    this.icebergCatalogWrapper = new IcebergCatalogWrapper(icebergConfig, false);
```

Comment:
Can you add a comment here to explain the reason why we hard code the `false` here?

Reply:
done